packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -2749,9 +2826,14 @@ function unwindUnitOfWork(unitOfWork: Fiber): void {
       returnFiber.deletions = null;
     }
 
-    // NOTE: If we re-enable sibling prerendering in some cases, here we
-    // would switch to the normal completion path: check if a sibling
-    // exists, and if so, begin work on it.
+    if (!skipSiblings) {
+      const siblingFiber = incompleteWork.sibling;
+      if (siblingFiber !== null) {
```

Comment:
`if (siblingFiber) ` will check for both not null and not undefined , so there is no need to explicitly check for not null